cost/kubecost/cluster/kubecost_cluster_rightsizing_recommendations.pt

```diff
@@ -0,0 +1,226 @@
+    var totalNodeCount = ds_cluster_sizing.totalNodeCount
+    var totalRAMGB = ds_cluster_sizing.totalRAMGB
+    var totalVCPUs = ds_cluster_sizing.totalVCPUs
+    var monthlyRate = ds_cluster_sizing.monthlyRate
+    var results = []
+    var optimalValue = 0
+    var tempResult
+    _.forEach(ds_cluster_sizing.recommendations, function(e, k) {
+      e.rec_strategy = k;
+      e.clusterId = clusterId;
+      e.clusterName = clusterName;  
+      if (e.clusterName == "" || e.clusterName.lenght === 0) {
+        e.clusterName = clusterId;
+      }
+      e.currencyCode = ds_get_currency.currencyCode
+      if (e.currencyCode == "" || e.currencyCode.lenght === 0) {
+        e.currencyCode = "USD";
+      }
+      e.totalNodeCount = totalNodeCount
+      e.totalRAMGB = totalRAMGB
+      e.totalVCPUs = totalVCPUs
+      e.monthlyRate = monthlyRate
+      e.strPools = JSON.stringify(e.pools);
+      if (recommendation_strategy.toLowerCase() == "optimal") {
+        if (e.monthlySavings > optimalValue) {
+          optimalValue = e.monthlySavings
+          e.monthlySavings = Math.round(e.monthlySavings * 100) / 100
+          tempResult = e
+        }
+      } else {
+        if (k == recommendation_strategy.toLowerCase()) {
+          e.monthlySavings = Math.round(e.monthlySavings * 100) / 100
+          results.push(e)
+        }
+      }
+    })
+    if (recommendation_strategy.toLowerCase() == "optimal") {
+      results.push(tempResult)
+    }
+  EOF
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "policy_kubecost_cluster" do
+  validate_each $ds_cluster_array do
+    summary_template "Kubecost Cluster Sizing Report"
+    escalate $report_volumes
+    check eq(0,1)
+    export do
+      field "clusterId" do
+        label "Cluster ID"
+      end
+      field "clusterName" do
+        label "Cluster Name"
+      end
+      field "rec_strategy" do
+        label "Recommendation Strategy"
+      end
+      field "totalNodeCount" do
+        label "Current Node Count"
+      end
+      field "totalRAMGB" do
+        label "Current Total RAM (GB)"
+      end
+      field "totalVCPUs" do
+        label "Current Total vCPUs"
+      end
+      field "currencyCode" do
+        label "Currency"
+      end
+      field "monthlyRate" do
+        label "Current Monthly Cost"
+      end
+      field "totalMonthlyCost" do
+        label "Recommended Monthly Cost"
+      end
+      field "monthlySavings" do
+        label "Monthly Savings"
+      end
+      field "nodeCount" do
+        label "Node Count"
+      end
+      field "strPools" do
+        label "Pools"
+      end
+    end
+  end
+end
+
+###############################################################################
+# Escalations
+###############################################################################
+
+escalation "report_volumes" do
```

Comment:
escalation declarations should be prefixed with `esc_`

```suggestion
escalation "esc_report_volumes" do
```

Reply:
Renamed escalation declaration!